scripts/generate_heatmap.R

```diff
@@ -376,15 +376,19 @@ overlay_gene_naming <- function(blast_results, gene_metadata_filepath) {
 #' @return ggplot heatmap
 #' @export
 plot_blast_heatmap <- function(blast_results) {
-  
+
+  # Just to be safe, only keep the top percent identity value for a query if there are multiple hits
```

Comment:
I realized that BackBLAST could be adapted in the future to output potential paralogs along with orthologs. Because of this, I made a couple minor changes to the viz script so that, if multiple hits (e.g., to both orthologs and paralogs) are included in the BLAST output file, the script will still work and will just plot the top hit based on percent identity.